src/utils/wallet-seed.js

```diff
@@ -111,3 +111,13 @@ async function deriveEncryptionKey(password, salt, iterations, digest) {
 export function lockWallet() {
   setUnlockedMnemonicAndSeed(null, null);
 }
+
+export function forgetWallet() {
+  localStorage.removeItem('walletIndex');
+  localStorage.removeItem('walletCount');
+  localStorage.removeItem('locked');
+  localStorage.removeItem('unlocked');
```

Comment:
```suggestion
  localStorage.removeItem('walletPrivateKeyImports');
```

Needed for private key imports.